azure-pipelines.yml

```diff
@@ -76,6 +76,6 @@ stages:
   jobs:
   - template: ./ci/templates/release.yml
     parameters:
-      job_condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/heads/CSUB-175-automate-github-release'))
+      job_condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/heads/main'))
```

Comment:
@dlebee this `startsWith()` condition doesn't make sense to me b/c this job is triggered on git tags, not branches.

I think it is a left-over from the original implementation and should be removed, right?

Reply:
yes